src/clusterfuzz/_internal/cron/load_bigquery_stats.py

```diff
@@ -174,6 +174,7 @@ def _load_data(fuzzer):
       try:
         logs.log("Uploading job to BigQuery.", job_body=job_body)
 
+        # pylint: disable=no-member
```

Comment:
btw If no-member only happens once or twice in a file, I put it alongside the issue (instead of on top), so it only is disabled for the line and not the rest of the file/function.

Reply:
This is all moved from existing code right? No need to fix up existing issues.

Reply:
Yes, it all move from existing code.